modules/acquia_cms_common/tests/src/Kernel/AcquiaCmsTelemetryTest.php

```diff
@@ -173,11 +205,14 @@ public function testAcquiaCmsTelemetryDataWithAcquiaCmsVersion(): void {
         "application_uuid" => "",
         "application_name" => "",
         "environment_name" => "",
+        'acsf_status' => FALSE,
+        'site_uri' => $this->sitePath,
+        'site_name' => $this->siteName,
         "starter_kit_name" => "acquia_cms_existing_site",
         "starter_kit_ui" => FALSE,
         "site_studio_status" => FALSE,
         'profile' => '',
-        "version" => "1.5.2",
+        'version' => '1.5.2',
```

Comment:
@rajeshreeputra can we keep double quotes or single quotes unique across all keys.